Make in Application Fails with fatal error: 'string.h' file not found

Hi, I was building in docker. make in `/sledge/runtime` works fine, however `make all` in `/sledge/applications `seems to have some linking problem
```
root@wanda0:/sledge/applications# make clean all
...
1 error generated.
In file included from CMSIS_5/CMSIS/DSP/Source/BasicMathFunctions/arm_sub_q31.c:29:
CMSIS_5/CMSIS/DSP/Include/arm_math.h:395:10: fatal error: 'string.h' file not found
#include <string.h>
         ^~~~~~~~~~
1 error generated.
In file included from CMSIS_5/CMSIS/DSP/Source/BasicMathFunctions/arm_sub_q7.c:29:
CMSIS_5/CMSIS/DSP/Include/arm_math.h:395:10: fatal error: 'string.h' file not found
#include <string.h>
         ^~~~~~~~~~
1 error generated.
cifar10/arm_nnexamples_cifar10.c:93:10: fatal error: 'stdio.h' file not found
#include <stdio.h>
         ^~~~~~~~~
1 error generated.
make[2]: *** [Makefile:49: cifar10.wasm] Error 1
make[2]: Leaving directory '/sledge/applications/wasm_apps/CMSIS_5_NN'
make[1]: *** [Makefile:414: dist/cifar10.wasm] Error 2
make[1]: Leaving directory '/sledge/applications/wasm_apps'
make: *** [Makefile:39: wasm_apps/dist/cifar10.wasm] Error 2
```
Do you possibly know why this happens? install_llvm.sh inside docker seems to run successfully though.

--- a/Dockerfile.x86_64
+++ b/Dockerfile.x86_64
@@ -80,7 +80,7 @@ RUN ./sledge/install_llvm.sh $LLVM_VERSION
 
 # WASI-SDK
 RUN curl -sS -L -O $WASI_SDK_URL && dpkg -i wasi-sdk_12.0_amd64.deb && rm -f wasi-sdk_12.0_amd64.deb
-ENV WASI_SDK=/opt/wasi-sdk
+ENV WASI_SDK_PATH=/opt/wasi-sdk
 
 # Create non-root user and add to sudoers
 ARG USERNAME=dev

--- a/applications/Makefile
+++ b/applications/Makefile
@@ -12,7 +12,7 @@ LDFLAGS=-shared -fPIC -Wl,--export-dynamic,--whole-archive -L../libsledge/dist/ 
 # LDFLAGS=-flto -fvisibility=hidden
 
 dist:
-	mkdir dist
+	mkdir -p dist
 
 .PHONY: all
 all: \

--- a/libsledge/Makefile
+++ b/libsledge/Makefile
@@ -21,7 +21,7 @@ CFLAGS := -fPIC -O3
 all: dist/libsledge.a
 
 dist:
-	mkdir dist
+	mkdir -p dist
 
 dist/%.o: src/%.c dist
 	clang ${CFLAGS} -c ${INCLUDES} -o $@ $<

